corehq/apps/data_cleaning/models/session.py

```diff
@@ -442,9 +442,14 @@ def can_select_all(self, table_num_records=None):
         # the most potentially expensive query is below:
         return num_records + self._get_num_unrecorded() <= MAX_RECORDED_LIMIT
 
-    def update_result(self, record_count, form_id=None, error=None):
+    def update_result(self, record_count, form_id=None, error=None, num_committed_records=None):
         result = self.result or {}
 
+        if 'num_committed_records' not in result and num_committed_records is not None:
+            result['num_committed_records'] = num_committed_records
+        elif 'num_committed_records' in result:
+            num_committed_records = result['num_committed_records']
+
```

Comment:
It seems like the logic is:
- if "num_committed_records" is in the result, use that
- otherwise, if `num_committed_records` isn't `None`, use that

Switching the ordering of the if statements seems like it would allow us to simplify the first `if` statement.

Reply:
thanks, good point
addressed here https://github.com/dimagi/commcare-hq/pull/36520/commits/c7100874a02e195ce2d6a3e305c734fbb341739a